docs/src/main/sphinx/admin/properties-catalog.md

```diff
@@ -0,0 +1,43 @@
+# Catalog management properties
+
+(prop-catalog-management)=
+
+## `catalog.management`
+
+- **Type:** {ref}`prop-type-string`
+- **Allowed values:** `static`, `dynamic`
+- **Default value:** `static`
+
+Trino reads catalog property files and configures available catalogs on server
+startup. When set to `dynamic`, catalog configuration can also be managed using
+[](/sql/create-catalog) and [](/sql/drop-catalog). Creating and dropping
+catalogs using these SQL commands will create new and delete existing catalog
+property files on all nodes.
```

Comment:
We also need to clarify that it requires read access to the filesystem on the coordinator and workers by the process running Trino.

Reply:
Also .. what happens if a new node is added via cluster scaling for example on k8s .. will it somehow get the catalog files created as necessary or not?

Reply:
I tested this locally in containers and workers get all the catalogs when joining the cluster (they're able to execute queries), but they don't write the files to disk. I updated relevant parts of the docs.